charts/jaeger/values.yaml

```diff
@@ -33,6 +33,13 @@ allInOne:
   #   }
   service:
     headless: true
+    allInOne:
```

Comment:
```suggestion
```

Since you use `.Values.allInOne.service.collector.otlp.grpc.name` above, this allInOne key is not needed here.